file-server/app.py

```diff
@@ -0,0 +1,149 @@
+        txt = llm.generate_text(
+            "Write a 1 minute info speech in plaintext for :" + data["text"]
+        )
+        return jsonify(
+            {
+                "user_id": user_id,
+                "text": txt,
+                "image_urls": [f"/static/{user_id}/images/{image}" for image in imgs],
+            }
+        )
+
+
+@app.route("/edittext", methods=["POST"])
+def edittext():
+    """
+    Takes in a text, generates a voiceover, and creates a video with the voiceover.
+    Returns the URL of the generated video.
+    """
+    if request.method == "POST":
+        data = json.loads(request.data.decode("utf-8"))
+        edit_text = data["text"]
+        user_id = data["user_id"]
+        vo_url = vo.generate_voiceover(
+            str(edit_text), f"./static/{user_id}/audio/output_voiceover.mp3"
+        )
+        mg.create_video(
+            vo_url,
+            image_folder=f"./static/{user_id}/images",
+            output_file=f"./static/{user_id}/movie/output_video_with_audio.mp4",
+        )
+        return jsonify(
+            {"video_url": f"/static/{user_id}/movie/output_video_with_audio.mp4"}
+        )
+
+
+@app.route("/images", methods=["POST", "GET"])
+def images():
+    """
+    Takes user_id from json body and returns the urls of the images (from local storage).
+    If it doesn't exist returns an error message.
+    """
+    if request.method == "POST":
+        json_data = json.loads(request.data.decode("utf-8"))
+        user_id = json_data["user_id"]
+        if os.path.exists(f"file-server/static/{user_id}/images"):
+            imgs = os.listdir(f"./static/{user_id}/images")
+            return jsonify(
+                {
+                    "user_id": user_id,
+                    "image_urls": [
+                        f"/static/{user_id}/images/{image}" for image in imgs
+                    ],
+                }
+            )
+        else:
+            return jsonify({"error": "Images not found."})
+
+
+@app.route("/voiceover", methods=["POST"])
+def voiceover():
+    """
+    Takes user_id from json body and returns the url of the voiceover (from local storage).
+    If it doesn't exist returns an error message.
+    """
+    if request.method == "POST":
+        json_data = json.loads(request.data.decode("utf-8"))
+        user_id = json_data["user_id"]
+        if os.path.exists(f"file-server/static/{user_id}/audio/output_voiceover.mp3"):
+            return jsonify(
+                {
+                    "user_id": user_id,
+                    "voiceover_url": f"/static/{user_id}/audio/output_voiceover.mp3",
+                }
+            )
+        else:
+            return jsonify({"error": "Voiceover not found."})
+
+
+@app.route("/movie", methods=["POST"])
+def movie():
+    """
+    Takes user_id from json body and returns the url of the video (from local storage). If it doesn't exist returns an error message.
+    """
+    if request.method == "POST":
```

Comment:
Why are we double checking this? It is already asserted as POST here:

`@app.route("/movie", methods=["POST"])`